ngraph/python/src/ngraph/utils/node_factory.py

```diff
@@ -82,6 +84,9 @@ def _arguments_as_outputs(arguments: List[Union[Node, Output]]) -> List[Output]:
             if issubclass(type(argument), Output):
                 outputs.append(argument)
             else:
+                log.warning("Op arguments were passed as Node, please avoid passing arguments in "
```

Comment:
Do we have a mechanism to mark this behavior as deprecated? If I understand correctly, we just introduced a runtime warning here.

Reply:
@ilyachur AFAIK there is no such thing. We can either use a python package called `deprecation` ([this one](https://deprecation.readthedocs.io/en/latest/#:~:text=To%20mark%20a%20function%20or,you%20set%20on%20deprecated()%20.)) or create our own decorator and put it inside `ngraph/decorators.py`. I would recommend creating our own decorator so we do not add another pip dependency. We can emulate a behavior of previously mentioned package. That means using some kind of versioning with `__version__ ` or just name of the release will provide accurate warning massage and even block from using the function if the version has been bumped up.
What you think about this?

Reply:
I think that this question about implementing  a new deprecation mechanism would be better suited for our tech sync, you can propose such topic to be discussed. To conclude, the answer to Ilya's original question is that at this moment there is no such mechanism present, and as such, I'm resolving this conversation for now.